### PR TITLE
add train images log

### PR DESCRIPTION
**Pull request by melisandeteng**
fixed typo in networks
Added train image log - at the same frequency as the test images

### CODE REVIEW

File: models/mask_generator.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def save_test_images(self, test_display_data, curr_iter):
             save_images.append(save_real_mask_seg)
             save_images.append(save_real_mask)
         write_images(
-            save_images, curr_iter, comet_exp=self.comet_exp, store_im=self.store_image
+            save_images, curr_iter, comet_exp=self.comet_exp, store_im=self.store_image, is_test = is_test
```
Comment (vict0rsch) [inline models/mask_generator.py:308]: Line is too long, I don't think `black` would let you do that ;)

File: shared/feature_pixelDA.yml [hunk 2]
```diff
@@ -119,7 +119,7 @@ model:
 
 
 comet:
-  workspace: "sunandr"
+  workspace: "melisandeteng"
```
Comment (vict0rsch) [inline shared/feature_pixelDA.yml:122]: haha now with the new command line args in master there's no need to fight for that anymore!

File: train.py [hunk 3]
```diff
@@ -69,9 +69,16 @@
...
     val_iter = iter(val_loader)
+    
     test_display_images = [
         Dict(val_iter.next()) for i in range(opts.comet.display_size)
     ]
+    
+    if opts.train.save_im:   
+        train_iter = iter(train_loader)
+        train_display_images = [
+            Dict(val_iter.next()) for i in range(opts.comet.display_size)
```
Comment (vict0rsch) [inline train.py:80]: 1 thing that bugs me with this method is why even ask for that many batches if only the first image of the batch is going to be used in test_networks anyway?

File: utils.py [hunk 4]
```diff
@@ -178,8 +178,10 @@ def write_images(
     image_grid = image_grid.permute(1, 2, 0).cpu().detach().numpy()
 
     if comet_exp is not None:
-        comet_exp.log_image(image_grid, name="test_iter_" + str(curr_iter))
-
+        if is_test:
+            comet_exp.log_image(image_grid, name="test_iter_" + str(curr_iter))
+        else: 
+            comet_exp.log_image(image_grid, name="train_iter_" + str(curr_iter))
```
Comment (vict0rsch) [inline utils.py:184]: you should always specify `step=` in the comet logs ; I know it works fine without it but sometimes it's messed up for some unknown reason, where you'd see thousands of steps with missing data. And this prevents that

File: models/mask_generator.py (hunk 1 above)
Comment (melisandeteng) [reply models/mask_generator.py:308]: Oh no you caught me! I edited in jupyterlab :)

File: utils.py (hunk 4 above)
Comment (vict0rsch) [reply utils.py]: I fixed that

**Pull request merged**